src/components/Book.css

```diff
@@ -0,0 +1,66 @@
+.book_container {
+  background-color: white;
+  margin-bottom: 10px;
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  max-width: 600px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.book-header {
+  display: flex;
+  flex-direction: column;
+}
+
+.book-progress {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
```

Comment:
- [ ]  As stated in the project requirement **Your final design should match Zeplin's design (this includes all elements + the same styling)** So Kindly I recommend you to book progress section to contain 'A circle update status icon' Just like the Given design so Kindly Fix it .

- The Zeplin's design Book progress section Look  likes the below : 
![book](https://github.com/MPDADDY/bookstore/assets/93598189/11698056-dd55-4e9d-84e1-7bf5a9dc24a7)

- Your Book Progress design Look likes this : 
![book](https://github.com/MPDADDY/bookstore/assets/93598189/3e9a8c83-96c8-494f-a334-3cbcbf15c0fc)